cypress.json

```diff
@@ -1 +1,5 @@
-{ "defaultCommandTimeout": 8000 }
+{
+  "baseUrl": "https://qa-react-conduit.herokuapp.com/",
```

Comment:
env

Reply:
This doesn't go to .env.... cypress use it to get de baseURL to their commands... like cy.visit('/')
![image](https://user-images.githubusercontent.com/43115836/123439372-121c5200-d5a8-11eb-9b97-313d49c16b0a.png)